lib/widgets/rules.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/material.dart';
+import 'package:tic_tac_toe/styles/colors.dart';
+
+import '../model/rules.dart';
+
+/// This widget is a text container.
+///
+/// It shows game rules.
+class GameRulesView extends StatelessWidget {
+  const GameRulesView({super.key});
+
+  final double lineSpacing = 15;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      constraints: const BoxConstraints(maxWidth: 400),
+      child: ExpansionTile(
+        collapsedBackgroundColor: AppColors.rules.title,
+        childrenPadding:
+            const EdgeInsets.symmetric(vertical: 10, horizontal: 15),
+        title: Text(
+          'Game rules',
+          style: Theme.of(context).textTheme.titleLarge,
+        ),
+        children: [
+          Column(
```

Comment:
How about adding introduction like this:

'For each board you can choose one of the rules variation to play:' 

And we can add fourth option: 

Wild
Choose your number of pieces to get in-a-row, and decide if getting there is winning (positive game) or losing (negative or misere).

Reply:
https://user-images.githubusercontent.com/64236396/235771769-2c08ca59-b86a-4624-9814-24fb76051971.mov